index.html

```diff
@@ -107,6 +107,74 @@ <h3>
       </dd>
     </dl>
   </section>
+  <section id="RTCVideoReceiverStats-dict*">
+    <h3>
+      <dfn>RTCVideoReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCVideoReceiverStats {
+      unsigned long numFreezes;
+      unsigned long numPauses;
+      double totalFreezesDuration;
+      double totalFramesDuration;
+      double sumOfSquaredFramesDuration;
+};</pre>
+    <dl data-link-for="RTCVideoReceiverStats" data-dfn-for="RTCVideoReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>numFreezes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long">unsigned long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video freezes experienced by this receiver. It is a
+          freeze if frame duration, which is time interval between consecutively
+          rendered frames, is 3 times higher than the average frame duration.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>numPauses</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video pauses experienced by this receiver. Video is
+          considered to be paused if time passed since last received packet
+          exceeds 5 seconds.
```

Comment:
If the transceiver is inactive or sendonly, does this still get updated? Is an inactive receiver "paused"? Same question for numFreezes. Would an inactive receiver that then becomes active (perhaps 5 minutes later) have a freeze of 5 minutes?

Reply:
Can you get RTCVideoReceiverStats for sendonly client? If yes, then all these metrics will be set to 0 (no frames were rendered).

If deactivation is accompanied with removal of receiving video stream then inactive interval won't be considered as a freeze (all recv stats is reset when received video stream is destroyed).

Reply:
A transceiver always has a receiver, whether it's sendonly or not (the sendonly / sendrecv / recvonly value can change over the lifetime of the transceiver, and this won't reset stats.
It makes sense to me that the freeze duration stops updating the moment the transceiver is set to sendonly or inactive.

Reply:
So what if it changes from inactive to recvonly or sendrecv. Would "time since last received packet" be "time since direction changed" or "time since the last received packet, not counting the first packet that arrives after direction changes from inactive or sendonly to sendrecv or recvonly"?

Reply:
If video receiver is never destroyed then it all depends on duration of interval of silence. If that lasts for at least 5 seconds than it will be counted as pause. Otherwise, it it lasts for longer than 3*avg_frame_duration, then it will be counted as freeze.

> Would an inactive receiver that then becomes active (perhaps 5 minutes later) have a freeze of 5 minutes?

Now I can answer this. Yes, pause (not freeze) duration will be 5 mins.

> So what if it changes from inactive to recvonly or sendrecv. Would "time since last received packet" be "time since direction changed" ...

The first inactive interval won't be counted neither as freeze nor pause. "time since last received packet" is literally what it says. If there was active interval before inactive (i.e. active -> inactive -> active) then last received packet is a packet from previous active interval (until replaced with newly received packet)